dotCMS/src/main/java/com/dotcms/experiments/business/RegexUrlPatterStrategy.java

```diff
@@ -21,7 +21,11 @@
  * - If the page is a Detail Page for a Content Type then we need to take account that the URL can be different.
  */
 abstract class RegexUrlPatterStrategy {
```

Comment:
@freddyucv This is a candidate to be an interface.

Reply:
done https://github.com/dotCMS/core/pull/25254/commits/f907cd967b1dc06ef0cd95a90285f0d1753ef943#diff-0e580f38c58875e09d3247c7fd86090d1237652e2c5e257a53a8a27a799fbb6aR23